gramps/gen/plug/utils.py

```diff
@@ -173,8 +173,11 @@ def is_file(self, name):
         """
         return os.path.split(name)[1]
 
-def available_updates():
-    whattypes = config.get('behavior.check-for-update-types')
+def urlopen_no_cert(URL):
```

Comment:
Since it's no-cert only on mac this isn't quite right as a function name. How about urlopen_maybe_no_check_cert() or urlopen_no_check_cert_on_mac()?